### PR TITLE
Feature: WRITE_KZG_COMMITMENT hint

### PR DESCRIPTION
**Pull request by odesenfans**
This implementation uses a default (panicking) implementation of the KZG
commitment computation as we do not know how these must be computed.

Issue Number: N/A

## Type

- [x] feature
- [ ] bugfix
- [ ] dev (no functional changes, no API changes)
- [ ] fmt (formatting, renaming)
- [ ] build
- [ ] docs
- [ ] testing

## Breaking changes?

- [ ] yes
- [x] no

### CODE REVIEW

**Review by maciejka — COMMENTED** [review]
In what sense we do not know what needs to be computed? Python implementation is not clear?
In case it is not already known: https://community.starknet.io/t/data-availability-with-eip4844/113065

File: src/execution/helper.rs [hunk 1]
```diff
@@ -29,7 +30,10 @@ type StorageByAddress = HashMap<Felt252, OsSingleStarknetStorage<DictStorage, Pe
 
 /// Maintains the info for executing txns in the OS
 #[derive(Debug)]
-pub struct ExecutionHelper {
+pub struct ExecutionHelper<KZG>
```
[the remark this answers is not in the document]
Comment (odesenfans) [reply src/execution/helper.rs:33]: Exactly, which is why a generic makes sense here. It's specified as a callback for some reason in the Python code, hinting that several implementations may be used. On the other hand, it's Rust, so it's better if we can get this indirection to be compile-time. This led me to add this as a generic. I'd be happy with having only one implementation as well but I could not find the implementation of the KZG computation anywhere.

[the remark this answers is not in the document]
Comment (maciejka) [reply src/execution/helper.rs:33]: Over-engineering imho. Even if there are multiple implementations of kzg there is no need to have multiple ExecutionHelpers using different implementations at the same time.I consult Ariel, maybe I am missing something.

[the remark this answers is not in the document]
Comment (maciejka) [reply src/execution/helper.rs:33]: Python uses: https://pydigger.com/pypi/ckzg.
It seems there is a pure rust version: https://github.com/grandinetech/rust-kzg.
Another thing to consult with Ariel.

**Comment by odesenfans** [comment]
Superseded by #304.

**Pull request closed without merging**